ground/src/main/java/com/google/android/ground/model/map/Map.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.android.ground.model.map
+
+import com.google.android.ground.model.geometry.Coordinate
+import com.google.common.collect.ImmutableList
+
+interface Map {
```

Comment:
These classes seem to be abstractions of view behaviors rather than the domain model, and as such I'd think they'd live in the view layer. Please cmiiw!

Reply:
Yes, that makes sense -- I was trying to clean up the organization in our ui.map section a bit -- would it make sense to have:

`ui.map.model` -- or do we feel that's overkill?

Reply:
I'll just move these back into `ui.map` for now. We can revisit this later.